### ISSUE TITLE
Add extra input

### ISSUE THREAD

**Issue by rhoddan** [niklaswall/node-red-contrib-light-scheduler#4]
Would be very good to have an additional input (parallell to current input) with an external ”dark” trigger. For example if I have an own external lux meter I can trigger false or true and that means I can avoid light on if it is too light outside (via my external meter)

**Comment by bbkinghun** [comment]
This is the best solution.

**Comment by niklaswall** [comment]
Yes, that is planned. The idea is to look for a msg.lux value and use that together with a threshold setting to determine "if it is dark enough" to enable the output.

**Comment by Mikro007** [comment]
When will this go live?
